### DISCUSSION TITLE
More arrow heads

### DISCUSSION THREAD

**Discussion by tsram90** [jgraph/drawio#2749] [Ideas]
There are different types of arrow heads used in many places. One I am missing a lot in Draw is half arrow with a perpendicular dash.
![image](https://user-images.githubusercontent.com/31439548/162750175-5b6f656b-4f9f-45e2-b784-2bfd606dd8a7.png)
It is widely used in Bond Graphs, which is a modelling method. 

Even better would be a way to make and store custom Arrow heads.

**Comment by chipbite** [comment]
Related; would love to get arrow to bubble / open circle. 
---->O
Circle on the "box"/line of the connected shape.
